### PR TITLE
Update with horde upstream

### PR DESCRIPTION
**Pull request by boekhorstb1**

### CODE REVIEW

File: .horde.yml [hunk 1]
```diff
@@ -30,36 +30,36 @@ authors:
...
+  release: alpha
+  api: alpha
 license:
   identifier: ASL
   uri: http://www.horde.org/licenses/apache
 dependencies:
   required:
-    php: ^5.3 || ^7
-    pear:
-      pear.horde.org/horde: ^5
```
Comment (midahp) [inline .horde.yml]: @boekhorstb1 is your branch maybe not rebased correctly? For example I don't see these entries in the target branch: https://github.com/maintaina-com/ingo/blob/FRAMEWORK_6_0/.horde.yml

File: composer.json [hunk 2]
```diff
@@ -21,44 +22,52 @@
...
     ],
-    "version": "4.0.0",
-    "time": "2017-11-11",
+    "time": "2021-10-08",
     "repositories": [
         {
-            "type": "pear",
-            "url": "https://pear.horde.org"
+            "type": "composer",
+            "url": "https://horde-satis.maintaina.com/"
```
Comment (midahp) [inline composer.json]: same here. I think your branch is not rebased correctly. Try to do this first

**Review by midahp — COMMENTED** [review]
your branch need a rebase first, so the diff here is readable

File: composer.json (hunk 2 above)
Comment (boekhorstb1) [reply composer.json]: ah thanks, ok will check

Comment (boekhorstb1) [reply composer.json]: repos diverged, I resolved the conflict in favor of upstream

**Pull request merged**